faust/transport/drivers/aiokafka.py

```diff
@@ -1482,7 +1482,7 @@ async def _create_topic(
                 topic,
                 partitions,
                 replication,
-                loop=asyncio.get_event_loop_policy().get_event_loop(),
+                loop=self.loop,
```

Comment:
This is the only part of the merge I have issues with. I'm surprised we called `asyncio.get_event_loop_policy().get_event_loop()` here to begin with. `loop=loop` definitely makes more sense, though.

Reply:
I wasn't really sure if it should be `self.loop or asyincio.get_event_loop_policy().get_event_loop()` instead of only `self.loop`

Reply:
`self.loop` makes more sense. I'll review usage of `asyncio.get_event_loop_policy().get_event_loop()` before the next release. The motivation for using it stems originally from https://stackoverflow.com/a/48605024/9852671 while we were updating `mode-streaming`.